packages/voicechat-server/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
it seems like this package doesn't join the project reference (ts) neither pnpm workspace. I think it's good cause this subpackage doesn't really co-operate with other packages in JS or type level. I guess we should have a new folder for this kind of packages instead of `packages/`. How do you think cc @septs @guanbinrui

Reply:
We could create a folder like "ext" or "external" and put it there

Reply:
i think, need be split to https://github.com/DimensionDev/voicechat-server
CC @yisiliu